docker-compose-prod.yml

```diff
@@ -3,8 +3,8 @@ services:
   app:
     build: .
     environment:
-      - S3_EXTENSIONS_BUCKET_HOST=brave-core-ext.s3.brave.com
-      - S3_EXTENSIONS_BUCKET_HOST_TOR=tor.bravesoftware.com
+      - S3_EXTENSIONS_BUCKET_HOST=brave-ext.s3.brave.com
```

Comment:
think we can skip changing this file

Reply:
ok, then should I remove the new bucket?

Reply:
the new buckets are fine as `.s3.brave.com` is a Fastly service that we can point to them

Reply:
so we are going to update fastly manually to point to the new buckets once we sync content over to new buckets?

Reply:
not manually, but change here to point to new bucket https://github.com/brave/devops/blob/master/terraform/go_updater/staging/go_updater.tf#L206

Reply:
and here https://github.com/brave/devops/blob/master/terraform/website/tor.bravesoftware.com/production/main.tf#L28

Reply:
these old is replaced here https://github.com/brave/devops/pull/3175/files#diff-2bf05b7a43cf1a71b7c5a9c59dff11c3R12-R15, so new bucket -> fastly `brave-ext.s3.brave.com`

Reply:
ok, we can change that if you want, my preference was for not revealing our bucket names by the Fastly services

Reply:
if that's the case, feel free to suggest new fastly end point for all the 4 new buckets, I will update the other PRs

Reply:
let's see what @jumde thinks. the changes for the dev ones look good so you can apply and test with those for now

Reply:
@linhkikuchi - changes look good, but we need to ensure that the endpoints are keyed correctly. cc: @bkero

Reply:
For this change, you'll also need to update the `S3_EXTENSIONS_BUCKET`: https://github.com/brave/devops/search?q=S3_EXTENSIONS_BUCKET&unscoped_q=S3_EXTENSIONS_BUCKET

Reply:
I'd still suggest to keep the same endpoints for production and just change the bucket in the Fastly service

Reply:
@mihaiplesa Our shared fastly module will create bucket, then point it to fastly. So if we use old endpoints and apply in production, it will create new empty production bucket and point to old endpoint. As we do not want any service disruption during bucket sync process, only way we can do it is just deploy with new endpoint, then after we've done syncing all buckets, re-apply with old endpoint to switch. Are you OK with this? (If we are doing to keep old endpoints for production, may as well keep old endpoints for dev also, this way, our bucket name and endpoints are different to hide our bucket name)

Reply:
depends on the hardcodings in other places as @jumde pointed. if we can avoid having to change in many places that would be preferable. not sure we have to sync the bucket or just run every pipeline to publish new versions to the new one.

Reply:
ok, I will keep all current fastly endpoints, and switch them to new buckets after syncing the buckets (to make sure they have at least current versions before we switch). I can see that ios has hardcoded fastly endpoint alias.

Reply:
there's also ways to use terraform import for some of this